cryptofeed/exchange/binance_futures.py

```diff
@@ -118,5 +118,143 @@ async def message_handler(self, msg: str, conn: AsyncConnection, timestamp: floa
+        await self.callback(ORDER_INFO,
+                            feed=self.id,
+                            symbol=self.exchange_symbol_to_std_symbol(msg['o']['s'].upper()),
+                            status=status,
+                            order_id=msg['o']['i'],
+                            side=BUY if msg['o']['S'].lower() == 'buy' else SELL,
+                            order_type=msg['o']['ot'],
+                            timestamp=msg['E'],
+                            receipt_timestamp=timestamp,
+                            **data
+                            )
+
+    async def _account_update(self, msg: dict, timestamp: float):
+        """
+        {
+          "e": "ACCOUNT_UPDATE",                // Event Type
+          "E": 1564745798939,                   // Event Time
+          "T": 1564745798938 ,                  // Transaction
+          "a":                                  // Update Data
+            {
+              "m":"ORDER",                      // Event reason type
+              "B":[                             // Balances
+                {
+                  "a":"USDT",                   // Asset
+                  "wb":"122624.12345678",       // Wallet Balance
+                  "cw":"100.12345678",          // Cross Wallet Balance
+                  "bc":"50.12345678"            // Balance Change except PnL and Commission
+                },
+                {
+                  "a":"BUSD",
+                  "wb":"1.00000000",
+                  "cw":"0.00000000",
+                  "bc":"-49.12345678"
+                }
+              ],
+              "P":[
+                {
+                  "s":"BTCUSDT",            // Symbol
+                  "pa":"0",                 // Position Amount
+                  "ep":"0.00000",            // Entry Price
+                  "cr":"200",               // (Pre-fee) Accumulated Realized
+                  "up":"0",                     // Unrealized PnL
+                  "mt":"isolated",              // Margin Type
+                  "iw":"0.00000000",            // Isolated Wallet (if isolated position)
+                  "ps":"BOTH"                   // Position Side
+                }，
+                {
+                    "s":"BTCUSDT",
+                    "pa":"20",
+                    "ep":"6563.66500",
+                    "cr":"0",
+                    "up":"2850.21200",
+                    "mt":"isolated",
+                    "iw":"13200.70726908",
+                    "ps":"LONG"
+                 },
+                {
+                    "s":"BTCUSDT",
+                    "pa":"-10",
+                    "ep":"6563.86000",
+                    "cr":"-45.04000000",
+                    "up":"-1423.15600",
+                    "mt":"isolated",
+                    "iw":"6570.42511771",
+                    "ps":"SHORT"
+                }
+              ]
+            }
+        }
+        """
+        await self.callback(ACCOUNT_UPDATE,
```

Comment:
it doesnt look like you actually are returning any data here? Wouldnt you want to return information about the account here too?